pkg/wsman/cim/physical/memory.go

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  **********************************************************************/
 
+// Package physical facilitiates communication with Intel® AMT devices as PhysicalMemory is a subclass of CIM_Chip, representing low level memory devices - SIMMS, DIMMs, raw memory chips, etc.
```

Comment:
This doesn't quite read correctly.  Probably should be something like "Package physical facilitates communications with Intel(r) AMT devices to get the PhysicalMemory component as a subclass of CIM_Chip, representing..."